web/app/public/index.html

```diff
@@ -8,10 +8,16 @@
     <title>{{ .Title }}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/manifest.json">
     <link rel="shortcut icon" href="/favicon.ico" />
-    <meta property="description" content="{{ .Description }}" />
+    <meta name="description" content="{{ .Description }}" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <meta name="apple-mobile-web-app-title" content="{{ .Title }}">
+    <meta name="application-name" content="{{ .Title }}">
+    <meta name="theme-color" content="#f7f9fb">
```

Comment:
Sorry, I missed it during my first check: these should probably all have the same endpoint (either `/>` or `>`) for the sake of consistency.

No need to regenerate the static assets, just updating this file is sufficient.

Reply:
Hove I can see at now this [file](https://github.com/TwiN/gatus/blob/master/web/app/public/index.html) doesn't have full consistency and use both styles.  I can make all this file in one style, what style do you prefer (`/>` or `>`) ?

Reply:
I personally prefer `/>`, but I'm fine with either :)

Reply:
done